lib/associations/belongs-to-many.js

```diff
@@ -464,6 +464,10 @@ BelongsToMany.prototype.injectGetter = function(obj) {
       }
     }
 
+    if(options.hasOwnProperty('schema')){
+      model = model.schema(options.schema,options.schemaDelimiter || '');
```

Comment:
No need to add the default `|| ''` I think - An empty value should be handled further down